adapter/memquota2/dedup.go

```diff
@@ -105,7 +105,7 @@ func (du *dedupUtil) reapDedup() {
 	du.oldDedup = du.recentDedup
 	du.recentDedup = t
 
-	if du.logger.VerbosityLevel(4) {
+	if len(t) > 0 && du.logger.VerbosityLevel(4) {
```

Comment:
have we fixed this elsewhere yet?

Reply:
Looks like a bad merge somewhere along the way stomped the fix: https://github.com/istio/mixer/pull/1224 was merged but https://github.com/istio/mixer/pull/1228 undid it by mistake.